doc/DEVELOPER_FAQ.md

```diff
@@ -46,7 +46,7 @@ The comments given in source code to structure `struct overmap_special` explain
 1. Edit `data/json/bionics.json` and add your bionic near similar types. See `JSON_INFO.md` for a more in-depth review of the individual fields.
 2. If you want the bionic to be available in the game world as an item, add it to `item_groups.json`, and add a bionic item to `data/json/items/bionics.json`.
 3. Manually code in effects into the appropriate files, for activated bionics edit the `player::activate_bionic` function in `bionics.cpp`.
-4. For bionic ranged weapons add the bionic weapon counterparts to `ranged.json`, give them the `NO_AMMO` and `BIO_WEAPON` flags.
```

Comment:
Did you change the flag, or was that a typo?

Reply:
That was a typo. In the code there is a constant defined as ` flag_BIO_WEAPON( "BIONIC_WEAPON" )`:  https://github.com/CleverRaven/Cataclysm-DDA/blob/68e91cfe72880ca444a7d3cb9a2633541c61e615/src/bionics.cpp#L209
It did confused me initially. Probably a good idea to change it as well for consistency.